packages/services/build/standalone/default-create/src/lib.rs

```diff
@@ -97,7 +97,7 @@ async fn upload_build(
 			backend::upload::PrepareFile {
 				path: unwrap!(unwrap!(build.key.file_name(), "should have file name").to_str()).to_string(),
 				content_length: len,
-				multipart: false,
+				multipart: len > util::file_size::mebibytes(100),
```

Comment:
i updated upload prepare to automatically handle this https://github.com/rivet-gg/rivet/blob/eaee5a54741843a6630262c9aab5a455585577a9/packages/services/upload/ops/prepare/src/lib.rs#L255